dist/templates/decanter/components/media/media.twig

```diff
@@ -28,6 +28,12 @@
  * - media_caption: Optional caption for the media.
  */
 #}
+
+{# Set link to null if there is no content to prevent an empty link #}
```

Comment:
@pookmish this doesn't seem to work correctly. The other parts of the pull request  do work properly.  What would be the steps to get this one fixed? I have a ticket already for this, D8CORE-2529 which is what this below was supposed to fix.

Reply:
I don't know. Due to it's inheritance from decanter I avoid it. Decanter never plays nice for the things I want it to and I constantly fight it.

Reply:
> Decanter never plays nice for the things I want it to and I constantly fight it.

You know, you do have the ability to fix the things you don't like or at the very least, report them and create issue tickets.

Reply:
The spider web that is Decanter makes it very difficult for me to navigate and make any changes